tools/config.py

```diff
@@ -137,6 +139,7 @@ def parse_config_file():
     'CACHE',
     'PORTS',
     'COMPILER_WRAPPER',
+    'TSC',
```

Comment:
What do you think about skipping the config and just using `PATH` for now?  i.e. `utils.which('tsc'`)`?